corpus/templates/pages/sig.html

```diff
@@ -0,0 +1,52 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block title %}
+{{sig.name}}
+{% endblock %}
+
+{% block content %}
+<!-- Hero Section -->
+<!-- TODO: Add action button -->
+<section
+    class="hero min-h-screen min-w-screen bg-gradient-to-br from-sky-50 to-blue-300 dark:from-blue-950 dark:to-base-100">
+    <div id="particles-js" class="h-screen w-full"></div>
+    <div style="gap: 20px; justify-content: space-between; margin: auto;">
+    {% for society in societies_linked_to_sig %}
+        <a class="my-5" href="{{society.url}}">
+            {% if society.dark_image %}
+                <img src="{% static '/' %}{{ society.image }}" style="display: none; padding: 10px; height: 250px; width: auto;" alt="{{ society.name }} Logo" class="light-logo h-14 lg:h-24 inline" >
```

Comment:
Why are you using `{% static '/' %}{{ society.image }}` instead of directly using `{% static society.image %}`?

Reply:
Alright, did it this way